"Common Examples" fails to execute

This action fails when using the "Common Examples" example from the readme:

workflows/label.yml:
```yaml
# This workflow will triage pull requests and apply a label based on the
# paths that are modified in the pull request.
#
# To use this workflow, you will need to set up a .github/labeler.yml
# file with configuration.  For more information, see:
# https://github.com/actions/labeler/blob/master/README.md

name: Labeler
on: [pull_request]

jobs:
  label:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/labeler@v2
      with:
        repo-token: "${{ secrets.GITHUB_TOKEN }}"

```

labeler.yml:
```yaml
# Add 'repo' label to any root file changes
repo:
  - ./*

# Add '@domain/core' label to any change within the 'core' package
@domain/core:
  - package/core/*
  - package/core/**/*

# Add 'test' label to any change to *.spec.js files within the source dir
test:
  - src/**/*.spec.js

# Add 'source' label to any change to src files within the source dir EXCEPT for the docs sub-folder
source:
- any: ['src/**/*', '!src/docs/*']

# Add 'frontend` label to any change to *.js files as long as the `main.js` hasn't changed
frontend:
- any: ['src/**/*.js']
  all: ['!src/main.js']
```

yields following error in gh actions:

```
  Set up job5s
  Run actions/labeler@v21s
    ^
Run actions/labeler@v2
##[error]YAMLException: end of the stream or a document separator is expected at line 6, column 1:
    @domain/core:
    ^
##[error]end of the stream or a document separator is expected at line 6, column 1:
    @domain/core:
    ^
  Complete job
```

I'm also seeing the examples fail, but differently.

```
source:
- any: ['src/**/*', '!src/docs/*']
```

run in isolation fails with:

```
##[error]TypeError: glob pattern string required
##[error]glob pattern string required
```

> I'm also seeing the examples fail, but differently.
> 
> ```
> source:
> - any: ['src/**/*', '!src/docs/*']
> ```
> 
> run in isolation fails with:
> 
> ```
> ##[error]TypeError: glob pattern string required
> ##[error]glob pattern string required
> ```

I didn't mention this in my issue but the "any" and "all" directives are not working for me either

For anyone facing the issue that @kepikoi mentioned (using @domain/core):
It seems that the `@` symbol is causing the error in the .yml parsing. Just remove the `@` if you are getting this error:

```
##[error]YAMLException: end of the stream or a document separator is expected at line 6, column 1:
    @domain/core:
    ^
```

--- a/README.md
+++ b/README.md
@@ -66,7 +66,7 @@ repo:
 - '*'
 
 # Add '@domain/core' label to any change within the 'core' package
-@domain/core:
+'@domain/core':
 - package/core/*
 - package/core/**/*
 

